g3w-admin/core/api/base/views.py

```diff
@@ -586,7 +590,13 @@ def response_data_mode(self, request, export_features=False):
                     continue
 
             # sort values
-            values.sort()
+            if ('fformatter' in self.request_data
+                and 'ordering' in self.request_data
+                and self.request_data['fformatter'] in self.request_data['ordering']):
+                rev = True if self.request_data['ordering'].startswith('-') else False
+                values.sort(reverse=rev, key=lambda e: e[1])
+            else:
+                values.sort()
```

Comment:
> The sorting of results is made by `str.sort()` python function

@wlorenzetti does it work with mixed arrays?

![image](https://github.com/g3w-suite/g3w-admin/assets/9614886/2c6d1df9-42e1-4273-b4ed-fa731f6e0f0c)

Reply:
The result for this API is  list of types that are identical to each other

Reply:
Ok, the case above is purposely unlikely (with all different types).

But the real question is, it should also handle null values?

For example:

```
"data": [
    [ "A106",  "Piazza Pietro Leopoldo" ],
    [ "A103",  None],
    [ "A102",  "Via Atto Vannucci" ],
    [ "A108",  "Via Giandomenico Romagnosi" ],
    [ "B002",  "Via Giovanni Lampredi" ],
    [ "A101",  None],
    [ "A107",  "Via Ubaldo Montelatici" ]
  ]
```

Also for the previous example the error was generated on the `NoneType` value.

Reply:
Ok I just make same tests.